src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -260,23 +277,13 @@ export class EthereumBlockchainService implements BlockchainService {
    */
   static increaseGasPricePerAttempt(
     estimate: BigNumberish,
-    attempt: number,
     previousGas: BigNumberish | undefined
-  ): BigNumber {
-    // Try to increase an estimated gas price first
-    const estimateBN = BigNumber.from(estimate)
-    const increase = estimateBN.div(10).mul(attempt) // 10% increase per attempt
-    const increaseEstimate = estimateBN.add(increase)
-
-    if (attempt == 0 || previousGas == undefined) {
-      return increaseEstimate
+  ): bigint {
+    if (previousGas == undefined) {
```

Comment:
so this is a behavior change to the gas logic.  Previously we would add an increasing amount of padding per attempt on top of the gas estimate we get from ethers.  So the first attempt we'd just use the gas estimate from ethers directly.  The second attempt we'd add 10% on top of the estimate, and the third time we'd add 20% on top of the estimate.
In this new version we're just always adding 10% on top of the estimate every time.

I don't feel strongly about which logic is better, but could we possibly separate the changes to our estimation logic from the code changes required to keep the same logic with upgrading to the new version of the ethers package, each into their own PR?